cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/sync/class-download-sync.php

```diff
@@ -249,7 +249,6 @@ public function download_asset( $attachment_id, $source = null ) {
 			// Generate signatures.
 			$this->sync->set_signature_item( $attachment_id, 'options' );
 			$this->sync->set_signature_item( $attachment_id, 'cloud_name' );
-			$this->sync->set_signature_item( $attachment_id, 'suffix' );
```

Comment:
Why are we removing this line? Is it relevant to the feature?

Reply:
@dugajean - Suffix was removed as a sync type and made inline with the upload.